projects/storefrontlib/src/events/events.module.ts

```diff
@@ -1,12 +1,11 @@
 import { NgModule } from '@angular/core';
 import { CartPageEventModule } from './cart/cart-page-event.module';
-import { PageEventModule } from './page/page-event.module';
 import { ProductPageEventModule } from './product/product-page-event.module';
 
 /**
```

Comment:
Can we remove this or is it another ticket?

Reply:
Not really sure. I don't mind removing it in this PR.